crates/ra_assists/src/handlers/remove_dbg.rs

```diff
@@ -126,4 +140,52 @@ fn foo(n: usize) {
             "dbg!(n.checked_sub(4))",
         );
     }
+
+    #[test]
+    fn test_remove_dbg_keep_semicolon() {
+        // https://github.com/rust-analyzer/rust-analyzer/issues/5129#issuecomment-651399779
```

Comment:
@flodiebold is this what you were referring to in [your comment](https://github.com/rust-analyzer/rust-analyzer/issues/5129#issuecomment-651399779)? I couldn't reproduce the with a comment above the dbg!, but this test showed an error that I fixed